config/routes.rb

```diff
@@ -14,6 +14,11 @@
   end
 
   draw :api
+  namespace :assets do
+    namespace :link_previews do
```

Comment:
Do you think we should namespace `link_previews` instead of name controller `track_previews`?